backends/arm/test/ops/test_cat.py

```diff
@@ -127,7 +127,6 @@ def test_cat_tosa_BI(self, operands: tuple[torch.Tensor, ...], dim: int):
 
     # TODO: Remove @unittest.expectedFailure when this issue is fixed in Regor
     @parameterized.expand(Cat.test_parameters)
-    @unittest.expectedFailure
```

Comment:
Do we want to stop these from getting lowered in the release/0.4 branch? 

i.e. comment from here too?
https://github.com/pytorch/executorch/blob/main/backends/arm/arm_partitioner.py#L38

Reply:
no they're working correctly on this version of the compiler, so i've basically moved the minimum forward i could to get a new enough regor and related fixes.

If you're happy to wait for another respin for the missed TODO I can do so in an hour or so, but I'd be tempted to just go ahead given this is optional functionality for the release.

Reply:
Oh sorry I can't see color apparently LOL